sources/core/Stride.Core.AssemblyProcessor/ComplexSerializerRegistry.cs

```diff
@@ -382,6 +384,25 @@ public static IEnumerable<SerializableItem> GetSerializableItems(TypeDefinition
             }
         }
 
+        static bool IsAccessibleThroughAccessModifiers(PropertyDefinition property)
+        {
+            var get = property.GetMethod;
+            var set = property.SetMethod;
+
+            if (get == null)
+                return false;
+
+            bool forced = property.CustomAttributes.Any(a => a.AttributeType.FullName == "Stride.Core.DataMemberAttribute");
+
+            if (forced && (get.IsPublic || get.IsAssembly))
+                return true;
+
+            if (get.IsPublic)
+                return set?.IsPublic == true || set == null && property.DeclaringType.Fields.Any(x => x.Name == $"<{property.Name}>k__BackingField");
```

Comment:
Would be good to add a comment here with an example of when a backing field is present.

Reply:
Will do